### PR TITLE
Bugfix/ds 22448 get service ds agent

### PR DESCRIPTION
**Pull request by harrisonchen201703**
- Fix issue for https://github.com/deep-security/ansible/issues/7
- Modify missing space before the wording "roles" in Sample README
- Why not using service module?
   1. Service module does not have status check function:
       https://github.com/ansible/ansible/issues/30172 For Service module not support status checking
   2. Will be implemented until the change being merged.
       https://github.com/ansible/ansible/pull/20177  For adding status into Service module

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ A sample playbook:
 ```yaml
 - hosts: all
   roles:
-    -role: deep-security.deep-security-agent
+    - role: deep-security.deep-security-agent
     action: deploy
```
Comment (lovemew67) [inline README.md]: Add indents in other lines.

File: tasks/install-linux.yml [hunk 2]
```diff
@@ -127,14 +127,9 @@
...
 - name: Get service 'ds_agent' status 
-  shell: "service ds_agent status"
-  register: result
-  become: true
-
-- name: Check service is on
-  fail:
-    msg: "Failed to start agent service"
-  when: (result.stdout | regex_search('(running)')) != "running"
+  service:
```
Comment (chchhsiao) [inline tasks/install-linux.yml]: I think we don't need to start the service again.
So, either remove the status check if it is not necessary, or ignore the warning might be a choice as service module didn't provide the function to check status.

File: README.md (hunk 1 above)
Comment (harrisonchen201703) [reply README.md]: Indents added.

File: tasks/install-linux.yml (hunk 2 above)
Comment (harrisonchen201703) [reply tasks/install-linux.yml]: Will ignore warning until service module provide better checking for service status

**Review by lovemew67 — APPROVED** [review]

**Review by chchhsiao — APPROVED** [review]

**Pull request merged**